pagopa/io-app
# chore(Cross): [IOAPPX-249] Update `build.gradle`

> [!warning]
> Depends on https://github.com/pagopa/io-app/pull/5506

## Short description
This PR updates the `build.gradle` file and removes a few workarounds which are not needed anymore after the upgrade to RN 0.70.

## List of changes proposed in this pull request
- Removes `exclusiveContent` section, added in [this PR](https://github.com/pagopa/io-app/pull/4160) to address an issue with `react-native`. Patched for version >`0.63` as indicated in the [related issue](https://github.com/facebook/react-native/issues/35210).
- Removes `mavenLocal()` repository which doesn't appear to be included by default in the `build.gradle` since a few `react-native` version;
- Removes `configurations.all` section which was added by [this PR](https://github.com/pagopa/io-app/pull/267/files) for a dependency, `react-native-camera`, which we don't use anymore.

## How to test
We should be able to build Android. Test it by running: 
```
cd android && ./gradlew clean && cd .. 
yarn run-android 
```
